app/components/ui/inline/Button/Button.tsx

```diff
@@ -40,21 +44,35 @@ export const Button = ({
     styles[`button--${size}`],
     styles[`button--${variant}`],
     styles[`button--arrow-${arrowVariant}`],
-
     {
       [styles["mobile-full-width"]]: mobileFullWidth,
       [`${styles["button--link"]} ${styles[`button--link-${variant}`]}`]: href,
     }
   );
 
+  const content = () => {
```

Comment:
I tend to like writing out any helper that renders JSX as a Component just to do the same thing every time. I'm not sure I care that strongly about that being consistent across an app. I guess I'm more curious to know how you decide to make this a function that returns JSX vs a functional Component like `<ButtonContent { ...props } />`.

Reply:
Oh good catch. This doesn't actually need to be a function so I will fix that.